addons/main/XEH_postInit.sqf

```diff
@@ -2,10 +2,35 @@
 
 [] call FUNC(arsenal_postInit);
 
+// Clear inventory of a container
+[QGVAR(clearAllCargo), {call FUNC(clearAllCargo)}] call CBA_fnc_addEventHandler;
+// Clear inventory of every backpack in a container
+[QGVAR(clearCargoBackpacks), {call FUNC(clearCargoBackpacks)}] call CBA_fnc_addEventHandler;
+
```

Comment:
These events try to optimize network usage by sending a single CBA Event then each machine running local commands.  
As opposed to 4 global commands for a single backpack when using
```
clearItemCargoGlobal _container;
clearWeaponCargoGlobal _container;
clearMagazineCargoGlobal _container;
clearBackpackCargoGlobal _container;
```
and 4 per backpack when clearing backpacks created inside a backpack.

It's necessary because there are pre-filled versions of backpacks for default inventory cargo for ammo bearer, medic, engineer, etc. If one of those is created inside a weaponholder, it will already have the default inventory cargo, and then we try to add its contents again, duplicating the items.  
Also affected is when such a backpack is emptied out and placed inside a unit's backpack, who then switches to chest and then to back. the pre-filled backpack created via `setUnitLoadout` will be filled.

Reply:
Fair enough. I didn't look to deeply into it on my initial review pass. Feel free to leave it as is.